docs/src/dplyr.md

```diff
@@ -0,0 +1,293 @@
+# DataFramesMeta.jl tutorial
+
+This is a port of the HarvardX series PH525x Genomics class tutorial on dplyr. View the original [here](https://genomicsclass.github.io/book/pages/dplyr_tutorial.html) and the source [here](https://github.com/genomicsclass/labs/blob/master/intro/dplyr_tutorial.Rmd).
+
+## What is DataFramesMeta.jl?
+
+DataFramesMeta.jl is a Julia package to transform and summarize tabular data. It provides a more convenient syntax to work with DataFrames from [DataFrames.jl](https://github.com/JuliaData/DataFrames.jl). For a deeper explanation of DataFramesMeta.jl, see the [documentation](https://github.com/JuliaData/DataFramesMeta.jl). 
+
+DataFramesMeta.jl is heavily inspired by R's `dplyr`. If you are familiar with `dplyr` this guide should get you up to speed with DataFramesMeta.jl. 
+
+## Why Is It Useful?
+
+Like dplyr, the DataFramesMeta.jl package contains a set of macros (or "verbs") that perform common data manipulation operations such as filtering for rows, selecting specific columns, re-ordering rows, adding new columns and summarizing data. 
+
+In addition, DataFramesMeta.jl contains a useful operation `@combine` to perform another common task which is the "split-apply-combine" concept. We will discuss that in a little bit. 
+
+## How Does It Compare To Using Base Functions in Julia and in DataFrames.jl?
+
+If you are familiar with Julia, you are probably familiar with base Julia functions such  `map`, and `broadcast` (akin to `*apply` in R). These functions are convenient to use, but are designed to work with `Vector`s, not tabular data.
+
+DataFrames provides the functions `select`, `transform`, and more to work with data frames. Unlike `map` and `broadcast`, these functions are designed to work with tabular data, but have a complicated syntax. 
+
+DataFramesMeta.jl provides a convenient syntax for working with the vectors in a `DataFrame` so that you get all the convenience of Base Julia and DataFrames combined. 
```

Comment:
```suggestion
DataFramesMeta.jl provides a convenient syntax for working with the vectors in a data frame so that you get all the convenience of Base Julia and DataFrames combined. 
```